src/components/Chart/index.d.ts

```diff
@@ -17,6 +17,9 @@ export interface ChartProps extends BaseProps {
     disableXAxisTickLabels?: boolean;
     disableYAxisTickLabels?: boolean;
     maintainAspectRatio?: boolean;
+    plugins?: Array<object>;
+    pluginsChartConf?: object;
+    pluginsDatasetConf?: object;
```

Comment:
remove this props from the API. the component has to be like this:

https://github.com/nexxtway/react-rainbow/issues/1626